### PR TITLE
kubenet: hook pod bandwidth resources up to shaper

### PR DESCRIPTION
**Pull request by dcbw**
@bprashanth @thockin Last bit for shaping.

### CODE REVIEW

**Comment by dcbw** [comment]
But let the hairpin mode stuff go through first, since this will conflict with that; I'll rebase early next week.

File: pkg/kubelet/network/kubenet/kubenet_linux.go [hunk 1]
```diff
@@ -286,3 +307,38 @@ func buildCNIRuntimeConf(podName string, podNs string, podInfraContainerID kubec
 		IfName:      network.DefaultInterfaceName,
 	}
 }
+
+var minRsrc = resource.MustParse("1k")
+var maxRsrc = resource.MustParse("1P")
+
+func validateBandwidthIsReasonable(rsrc *resource.Quantity) error {
```
Comment (freehan) [inline pkg/kubelet/network/kubenet/kubenet_linux.go]: nit: these two functions can be moved into bandwidth util.

**Comment by dcbw** [comment]
@bprashanth @freehan PTAL thanks!

**Comment by freehan** [comment]
LGTM

**Comment by dcbw** [comment]
Flake was https://github.com/kubernetes/kubernetes/issues/24619

**Comment by dcbw** [comment]
@k8s-bot test this issue: #24619

**Pull request merged**